Duplicati/Library/Main/ProcessController.cs

```diff
@@ -234,6 +234,35 @@ private void ActivateBackgroundIOPriority()
             }
         }
 
+        /// <summary>
+        /// Expose all filesystem attributes
+        /// </summary>
+        private static void ExposeAllFilesystemAttributes()
+        {
+            // Starting with Windows 10 1803, the operating system may mask the process's view of some
+            // file attributes such as reparse, offline, and sparse.
+            //
+            // This function will turn off such masking.
+            //
+            // See https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-rtlqueryprocessplaceholdercompatibilitymode
+
+            if (Platform.IsClientWindows)
+            {
+                try
+                {
+                    var currentPcm = Win32.RtlQueryProcessPlaceholderCompatibilityMode();
+                    if (currentPcm == Win32.PHCM_VALUES.PHCM_DISGUISE_PLACEHOLDER)
+                    {
+                        Win32.RtlSetProcessPlaceholderCompatibilityMode(Win32.PHCM_VALUES.PHCM_EXPOSE_PLACEHOLDERS);

```

Comment:
Just call the setter without the if, and remove the getter. Then we also don't need the import of the getter.

Reply:
Yep I can do that if you prefer. I will submit an update later today.  Thanks